arrocke/gloss-translation
# feat: import gloss data from old server

## What has changed

Actually implemented the logic for importing glosses from https://hebrewgreekbible.online

## Connected Issues

closes #60 


## QA Steps

1. Create a language or use an existing one
2. Go to the import page for your language
3. Select a language from the old site in the dropdown
4. Click "import glosses" and confirm the dialog
5. Wait for the import process to complete. It will probably take a few notes
6. Go to Genesis 1:1 or some other verse. Check that glosses were imported correctly
    - You may have to search for a verse with glosses. Many of the languages from the old site are incomplete

## Post-Deployment
